### ISSUE TITLE
Use AntCat for all ants

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/data#517]
[AntCat](https://antcat.org) is the leading database for ants, i.e. Formicidae.
They currently have over 4000 more species than [COL/ITIS](https://www.catalogueoflife.org/data/taxon/623T5) does today and include many fossils too:

> AntCat Formicidae
> Extant: 16 valid subfamilies, 38 valid tribes, 346 valid genera, 14.110 valid species
> Fossil: 6 valid subfamilies, 6 valid tribes, 171 valid genera, 781 valid species

> COL/ITIS
> genera: 359
> species: 10213

It is well maintained, NFS funded and has an API.

**Comment by mdoering** [comment]
@dhobern a pretty decent version of AntCat is now available on ChecklistBank using a CSV generated for Antweb and their API:
https://www.checklistbank.org/dataset/54937/imports

**Comment by mdoering** [comment]
ITIS:
<img width="1384" alt="image" src="https://user-images.githubusercontent.com/327505/226696534-29d4435d-bbbd-4a89-8ab6-61deddaac91a.png">

AntCat with roughly twice the number of species and genera: 
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/327505/226696713-33f5a7b8-0e06-4d2c-b65b-02047e12755e.png">

**Comment by mdoering** [comment]
According to ITIS it is from 2006:
https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=154193#null



[Deans et al. (July 2006)](https://www.itis.gov/servlet/SingleRpt/RefRpt?search_type=expert&search_id=expert_id&search_id_value=144)	 
 	Notes:	Consensus on Hymenoptera classification to family by: Andrew Austin, Denis Brothers, Matt Buffington, Andrew Deans, Gary Gibson, John Jennings, Norm Johnson, Johan Liljeblad, Michael Ohl, Lawrence Packer, Fredrik Ronquist, and Robert Wharton

**Comment by DaveNicolson** [comment]
Actually, the ITIS ant info is derived from the former Hymenoptera Name Server circa 2001+, when Norm Johnson was running it; the 2006 source is for the _hierarchy within all of Hymenoptera (including the families like Formicidae)_, not for any of the genera & species.

I would support COL moving to use AntBase if you can get a good version of their data. If you can get a working way of getting their _current_ data then you'd be able to keep it current, which would be ideal.

We are planning on updating the ants in ITIS using AntCat, as well, but it will take time. When I asked Norm about AntBase vs. AntCat he clearly sided with AntCat: "Antbase.org is a derivative of my database, so don’t bother with that. Use antcat.org."

**Comment by mdoering** [comment]
Thanks Dave. @myrmoteras also recommended AntCat as the best source to use.

**Comment by mdoering** [comment]
I am in contact with Brian Fisher from AntCat and it looks like we can at least get a weekly automated update of AntCat, maybe even daily.

**Comment by mdoering** [comment]
From an Antweb CSV export Brian gave to me - and which is created daily mostly.
And then adding on some additional information, mostly types, from the API.

**Comment by mdoering** [comment]
I have created a new [coldp generator](https://github.com/CatalogueOfLife/coldp-generator) here:
https://github.com/CatalogueOfLife/coldp-generator/blob/master/src/main/java/org/catalogueoflife/data/antcat/Generator.java

**Comment by dhobern** [comment]
Thanks @mdoering - this is great. Thanks for taking the lead on this. I'll raise this with the TG.

I've only just started looking at it (trying a few ants I know). A couple of suggestions for aspects that seem open to improvement.

https://www.checklistbank.org/dataset/54937/taxon/437190

* There seem to be some issues around the Remarks field - it's a chunk of mangled HTML - lots of open tags and none closed. Maybe they are closed and this is just how CLB renders it but stripping the tags out completely would be at least a step.
* The Published In field seems redundant and incomplete

https://www.checklistbank.org/dataset/54937/taxon/439637

* Similar for Remarks, but such a big block of unreadable text.

**Comment by mdoering** [comment]
remarks are the taxon history from antcat. I could remove tags in the future, but did not want to spend more time on this at this stage unless known to be useful for COL.

The reference is apparently a nested one where I have to properly export the in reference: https://antcat.org/references/132309

**Comment by mdoering** [comment]
I have fixed nested references and converted the html to plain text with markdown links. Importing now

**Comment by mdoering** [comment]
@dhobern @yroskov @DaveNicolson can we put AntCat on the TG agenda?
The data now gets updated weekly in ChecklistBank and seems to be in a pretty good state already to be adopted by COL

**Comment by DaveNicolson** [comment]
Absolutely!

**Comment by DaveNicolson** [comment]
If the data are in good shape (!) then I don't see a need to involve the TG, just replace the old ITIS data. That's my recommendation from ITIS' perspective. We will look into adopting this, as well. I didn't realize how far this had advanced!

**Comment by mdoering** [comment]
I haven't reviewed the data much, mostly syntactical. But there is quite some rich information provided with lots of remarks and many names with type information: https://www.checklistbank.org/dataset/54937/taxon/437190

**Comment by myrmoteras** [comment]
Yes. there is a lot of data which has been built on Bolton's catalogue from 1995, then antbase.org and Hymenoptera online  which has been based on all the ant literature we scanned and then, a Harvard University Press CD rom version of the catalogue, scraped on combined with antbase where all the errors have been removed and then antcat came up as another competitive project, with an official input from antbase. Antcat got off through US NSF funding for Eschmayers's fish catalogue, whereby Eschmayer bowed out at the last minute and the ant triumphed over the spiders to use the funding for building a community catalogue.
 So there is a a lot of data, including all name changes, synonyms, description of other morphs, etc., as well as comments to why changes occurred. 
 If I am right, they continualyl update, but not sure whether Bolton ships them still word documents with changes which then are being incorporated. 
 see eg Monomorium pharaonis https://antcat.org/catalog/440729 This includes also links to almost all the publications cited which are accessible as digital copies from antcat, based on antbase and Smithsoninan Atherton Seidal funded scans of the Agosti and Bill Brown Library.

Luckily, most of this is also available in CLB https://www.checklistbank.org/dataset/54937/taxon/440729, although it could be structured differently so that the references could be used, or the included taxonomic concepts. 

A very rich resource

**Comment by camiplata** [comment]
Ant Cat is now a source in COL
https://www.catalogueoflife.org/data/dataset/54937
